File: program (100).java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    CountSmall
//  Description :       Count Small Letters in the hardcoded String. 
//  Input :             String 
//  Output :            String
//  Author :            Aman Shaikh    
//  Date:               14/12/2024 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class StringDemo
{
    public int CountDiff(String str)
    {
        int iCnt = 0, i = 0, iCount = 0;
        while(i < str.length()) 
        {
            char ch = str.charAt(i);
            if(ch >= 'a' && ch <= 'z')
            {
                iCnt++;
            } 
            else if(ch >= 'A' && ch <= 'Z')
            {
                iCount++;
            }
            i++;
        }
        return (iCnt-iCount);
    }
}

class Assignment36C
{
    public static void main(String Arr[])
    {
        String s1 = "MarvellouS";
        StringDemo sobj = new StringDemo();
        int iResult = sobj.CountDiff(s1);
        System.out.println("Difference : "+iResult);
    }
}

File: program (100).cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    Sum                                        
//  Description :       Accept number from user and write a Recursive program and return summation of its digits. 
//  Input :             int 
//  Output :            int
//  Author :            Aman Shaikh    
//  Date:               15/12/2024 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Pattern
{
    public:
    int Sum(int iNo) 
    {
        int iNum = 0;
        static int iSum = 0; 

        if(iNo != 0) 
        {
            iNum = iNo % 10;
            iSum = iSum + iNum;
            iNo = iNo / 10;
            Sum(iNo);
        } 
        return iSum;
    }
};

int main()
{
    int iValue = 0, iRet = 0;

    cout<<"Enter the number : "; 
    cin>>iValue;

    Pattern pobj;
    iRet = pobj.Sum(iValue);

    cout<<"Summation is :\t"<<iRet;

    return 0;
}
